third_party_libs/devices/arm/st_microelectronics/stm32f1cube/stm32f1_repository.bzl

```diff
@@ -0,0 +1,66 @@
+load(":stm32f1_defs.bzl", "STM32_DEFINES")
+
+VERSIONS = {
+    "1.8.2" : {
+        "commit" : "441b2cbdc25aa50437a59c4bffe22b88e78942c9",
+        "sha256" : ""
```

Comment:
Please add a complete sha256 just to confirm determinism. If you leave this empty it should print out the sha256 when it fetches the remote reposistory.

Reply:
Got it. Will finish up the example this weekend.